app/controllers/app/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class App::CommentsController < App::BaseController
+  before_action :feedback_session, only: [:new, :create, :index]
+
+  def new; end
+
+  def create
+    comment = feedback_session.comments.create(body: params[:body])
+    if comment.persisted?
+      respond_with feedback_session, location: nil, status: :created
+    else
+      respond_with comment.errors, location: nil, status: :unprocessable_entity
+    end
+  end
+
+  def index
+    @comments = feedback_session.comments
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 16